ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml

```diff
@@ -36,6 +42,21 @@ tests:
     - chain: ipi-aws-pre
     - ref: cucushift-pre
     workflow: cucushift-e2e
+- as: e2e-aws-arm64-ipi
+  interval: 24h
+  steps:
+    cluster_profile: aws-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: arm-ci.devcluster.openshift.com
```

Comment:
We'd add the following two env variables to query the test cases in cucushift.
```
      E2E_RUN_TAGS: '@aws-ipi'
      TAG_VERSION: '@4.10'
```
Others could be needed. I'll give confirmation ASAP